src/loaders/OmeZarrLoader.ts

```diff
@@ -132,71 +236,41 @@ class OMEZarrLoader implements IVolumeLoader {
     const allmetadata = await data.attrs.asObject();
     // each entry of multiscales is a multiscale image.
     const imageIndex = imageIndexFromLoadSpec(loadSpec, allmetadata.multiscales);
-    const multiscales = allmetadata.multiscales[imageIndex].datasets;
-    const axes = allmetadata.multiscales[imageIndex].axes;
+    const multiscale: OMEMultiscale = allmetadata.multiscales[imageIndex];
+    const { datasets, axes } = multiscale;
 
     const axisTCZYX = remapAxesToTCZYX(axes);
-    const hasT = axisTCZYX[0] > -1;
-    const hasC = axisTCZYX[1] > -1;
+    this.hasT = axisTCZYX[0] > -1;
+    this.hasC = axisTCZYX[1] > -1;
```

Comment:
There probably should be a comment somewhere at the top of the IVolumeLoader class to say that loaders are stateful and should be kept alive.  If you call createVolume with one loader instance and then have to use a different one later, will hasT and hasC be initialized?  Maybe consider a function to lazily initialize hasT and has C just in case... not a strict requirement for this PR...